src/contracts/GPv2AllowanceManager.sol

```diff
@@ -2,12 +2,23 @@
 pragma solidity ^0.7.6;
 pragma abicoder v2;
 
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./libraries/GPv2SafeERC20.sol";
 import "./libraries/GPv2TradeExecution.sol";
 
 /// @title Gnosis Protocol v2 Allowance Manager Contract
 /// @author Gnosis Developers
 contract GPv2AllowanceManager {
+    using GPv2SafeERC20 for IERC20;
     using GPv2TradeExecution for GPv2TradeExecution.Data;
+    using SafeMath for uint256;
+
+    /// @dev A struct for direct transfers when settling a single order.
+    struct DirectTransfer {
```

Comment:
Namin nit:: maybe TargetTransfer is a better name, as it doesn't necessarily have to be direct (I could still transfer the funds to the settlement contract).

Reply:
I renamed the struct to be `Transfer` and the method to be `transferToTargets`. Let me know if you find this better.